code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/eventhub/EventHub.kt

```diff
@@ -136,14 +138,6 @@ internal class EventHub {
 
     init {
         registerExtension(EventHubPlaceholderExtension::class.java) {}
-        registerExtension(ConfigurationExtension::class.java) {
```

Comment:
Do we register the configuration extension in another place?

Reply:
I moved it to registerExtensions() API.